### PR TITLE
NodeRunner: fix OSX script. Add a delay before starting the next node.

### PR DESCRIPTION
**Pull request by adagys**
The sleep is needed so that Terminal opens every process in a new tab properly. Starting all nodes at the same time just leads to a single new tab being open..
I also think it's useful to start the nodes in order, e.g. with Raft or BFT notaries the first node has to fully initialize before all others do.

### CODE REVIEW

File: gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt [hunk 1]
```diff
@@ -22,11 +22,13 @@ fun main(args: Array<String>) {
         if (isNode(it)) {
             println("Starting node in $it")
             startedProcesses.add(runJar(nodeJarName, it, javaArgs))
+            Thread.sleep(1000)
```
Comment (exFalso) [inline gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt]: Is this the only way we can do this? If so, can we perhaps make this conditional on osx?

**Comment by mikehearn** [comment]
The code wasn't ported correctly from the shell version. Note that in the original code, a single AppleScript is created by concatenating code together and it has a "delay 0.5" in it. But in the new code, we execute multiple AppleScripts one after the other, with the "delay 0.5" line still there but it achieves nothing.

So the answer is yes, having a delay is the only way to do this.

**Comment by adagys** [comment]
Yes, and I think the purpose of the delay 0.5 was to wait until the new tab finishes opening (it's still valid here as well)

File: gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt (hunk 1 above)
Comment (adagys) [inline gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt]: Yeah can do

**Comment by adagys** [comment]
Tested it works on mac & Ubuntu

**Pull request merged**